intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -2864,6 +2890,7 @@ def scan_exchange_identifier(field):
     ('Vulnerable-ISAKMP', 'scan_isakmp', vulnerable_isakmp),
     ('Vulnerable-HTTP', 'scan_http', accessible_vulnerable_http),
     ('Vulnerable-Exchange-Server', 'scan_exchange', scan_exchange),
+    ('Vulnerable-SMTP', 'scan_smtp', vulnerable_smtp),
```

Comment:
```suggestion
    ('Vulnerable-SMTP', 'scan_smtp_vulnerable', vulnerable_smtp),
```
In our case the filename is ` 2021-08-12-scan_smtp_vulnerable-austria-geo.csv.zip`